hrm-service/docker-entrypoint

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env sh
 
-# Execute hrm-service migrations
-node ./db-migrate
+if [ -n "$HRM_RUN_MIGRATIONS" ]; then
```

Comment:
I don't know if this makes total sense. In practice I am only setting the `HRM_RUN_MIGRATIONS` env var on the poller, so the poller is responsible for running a single instance of migrations. instead of all of our 3 containers running them at the same time. I haven't done a deep dive into the table locking we're using for migrations, but I have run into weird behaviors with multiple instances running migrations that may take a LONG time and stepping on each other. 

If that is not a concern, then this should be removed so that all of the containers run migrations since there is a slight tradeoff where it is a small possibility that long running migrations may not finish by the time the HRM-service containers deploy. In k8s I would use a pre-init container to run migrations so that no others attempt deploy until the migrations are complete, but ECS doesn't support anything like that. It is kind of a pick your poison situation, I think. But am happy to revert this.

Reply:
Migrations *should* be atomic & idempotent, but limiting them to a single run per deploy will pre-empt strange hard to diagnose issues down the road.

My only slight concern is if it adversely affects the 'incompatibility window', i.e. how long an old HRM service node would potentially be running against a new, potentially incompatible version of the schema. But I suspect this window exists anyway so it's not a major concern (for this PR anyway), since it's not a new problem

Reply:
Also, I assume this poller will run the resource DB creation / migrations too when those get merged - don't see any reason not to

Reply:
The only way to decrease the window for the api part of hrm-service would be to only run migrations for the deployment of the regular service containers... which would cause the poller container to run against an out of date DB for some period of time.

In my experience, db migrations are always a pick your poison situation even in the best situation once you have more than one instance running unfortunately. The best way around major impact that I've found is to be wary of them and use multi-stage migrations when you know the underlying changes will have a potential major impact. Multi-stage meaning: Make your changes to support new code in one deploy and then quickly follow up with another deploy that include migrations to clean up the old db structures that are in use by old code.